program/src/processor.rs

```diff
@@ -188,20 +189,177 @@ impl Processor {
         let rent = Rent::get()?;
         check!(
             rent.is_exempt(mango_account_ai.lamports(), size_of::<MangoAccount>()),
-            MangoErrorCode::Default
+            MangoErrorCode::AccountNotRentExempt
         )?;
         check!(owner_ai.is_signer, MangoErrorCode::SignerNecessary)?;
 
-        let _mango_group = MangoGroup::load_checked(mango_group_ai, program_id)?;
+        let mango_group = MangoGroup::load_checked(mango_group_ai, program_id)?;
+        check!(
+            mango_group.num_mango_accounts <= mango_group.max_mango_accounts,
```

Comment:
After program upgrade, account creation will fail until SetMaxMangoAccounts was called (ah, maybe it has already been called?)

Why is this check even needed for the unclosable v0 accounts?

Reply:
Plan is to set this with a governance vote before deploying v3.3